tests/unit/src/test/scala/tests/codeactions/ImportMissingSymbolLspSuite.scala

```diff
@@ -118,13 +145,13 @@ class ImportMissingSymbolLspSuite
        |}
        |""".stripMargin,
     s"""|${ImportMissingSymbol.allSymbolsTitle}
-        |${ImportMissingSymbol.title("Future", "scala.concurrent")}
-        |${ImportMissingSymbol.title("Future", "java.util.concurrent")}
-        |${ImportMissingSymbol.title("Instant", "java.time")}
         |${ImportMissingSymbol.title("ListBuffer", "scala.collection.mutable")}
+        |${ImportMissingSymbol.title("Instant", "java.time")}
+        |${ImportMissingSymbol.title("Future", "java.util.concurrent")}
+        |${ImportMissingSymbol.title("Future", "scala.concurrent")}
+        |${CreateNewSymbol.title("ListBuffer")}
```

Comment:
Maybe not super important, but previously these suggestions seemed to be given from the top of the file down. These are now reversed starting from the bottom up. Do you think it'd be a good idea to keep them as is and shown top down?

Reply:
Good idea! Added sorting by a first diagnostic